app.js

```diff
@@ -1,34 +1,15 @@
 
 
 const readline = require('readline'); //console.info(readline);
-let userQuestions = readline.createInterface(process.stdin,process.stdout);
-let decided = readline.createInterface(process.stdin, process.stdout);
+// let input = readline.getPrompt();
```

Comment:
If we don't need it, delete it. there is no need to leave the comment here, it'll just clutter.